src/index.ts

```diff
@@ -318,6 +317,35 @@ export interface Theme {
   feedbackColor: string;
 }
 
+export interface GVLPurpose {
+  id: string
+  name: string;
+  description: string;
+  descriptionLegal: string;
+}
+
+export interface GVL {
+  purposes: {[key: string]:  GVLPurpose};
+  specialPurposes: {[key: string]:  GVLPurpose};
+  features: {[key: string]:  GVLPurpose};
+  specialFeatures: {[key: string]:  GVLPurpose};
+}
```

Comment:
leveraged for https://www.figma.com/file/rr4W8wjE0HwtgvI6j52RVY/Experience-Server?node-id=2518%3A8643. All vendor purposes listed even if customer only supports a subset

Reply:
One thing to note is as per TCF policies, we must represent their GVL "without modifications" (filtering allowed).